__tests__/index.test.js

```diff
@@ -90,3 +90,62 @@ describe('GET /api/articles/:article_id', () => {
 			});
 	});
 });
+
+describe('GET /api/articles', () => {
+	test('should return articles object with all articles', () => {
+		return request(app)
+			.get('/api/articles')
+			.then(({ body }) => {
+				body.articles.forEach((articles) => {
+					expect(articles).toMatchObject({
+						article_id: expect.any(Number),
+						title: expect.any(String),
+						topic: expect.any(String),
+						author: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						article_img_url: expect.any(String),
+						comment_count: expect.any(String),
+					});
+				});
+				expect(body.articles.length).toBe(13);
+				expect(body.articles.body).toBeUndefined();
+			});
+	});
+
+	test('should return articles object ordered by date', () => {
+		return request(app)
+			.get('/api/articles?sortby=created_at')
+			.then(({ body }) => {
+				expect(body.articles).toBeSortedBy('created_at');
+			});
+	});
+
+	test('allows the client to change the sort order to descending', () => {
+		return request(app)
+			.get('/api/articles?sortby=created_at&order=desc')
+			.then(({ body }) => {
+				expect(body.articles).toBeSorted('created_at', {
+					descending: true,
+				});
+			});
+	});
```

Comment:
This test (starting line 124) is the stronger test - you don't need to worry about the one starting on line 116